dss/api/bundles/__init__.py

```diff
@@ -288,6 +262,76 @@ def delete(uuid: str, replica: str, json_request_body: dict, version: str = None
 
     return jsonify(response_body), status_code
 
+def build_bundle_file_metadata(replica: Replica, user_supplied_files: dict):
+    handle = Config.get_blobstore_handle(replica)
+
+    time_left = nestedcontext.inject("time_left")
+
+    # decode the list of files.
+    files = [{'user_supplied_metadata': file} for file in user_supplied_files]
```

Comment:
pls avoid using python keywords as variable names.

Reply:
I saw that github highlighted it but it is really a python keyword?

Reply:
vim highlights it, but:
```
In [1]: import keyword                                                                                                                           
In [2]: keyword.iskeyword("file")                                                                                                                
Out[2]: False
```

Reply:
Nit: how about you change it to `_file` to make the machine happy?

Reply:
changed